src/System.Private.ServiceModel/src/System/ServiceModel/ActionNotSupportedException.cs

```diff
@@ -2,27 +2,26 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
 using System.Diagnostics.Contracts;
 using System.Runtime.Serialization;
 using System.ServiceModel.Channels;
 
 namespace System.ServiceModel
 {
+    [Serializable]
     public class ActionNotSupportedException : CommunicationException
     {
         public ActionNotSupportedException() { }
         public ActionNotSupportedException(string message) : base(message) { }
-        public ActionNotSupportedException(string message, Exception innerException) : base(message, innerException) { throw new PlatformNotSupportedException(); }
```

Comment:
Presumably this was intentional and the base constructor does everything necessary? Just double checking since the repetitive pattern in this PR is removing PNSEx from serialization constructors, which this is not.

Reply:
I checked the history of the original change and this looks like it was a mistake where the PNSE was thrown from the wrong method, and nobody noticed. How it is now is correct. WCF has lots of exceptions which provide no additional functionality and are there purely to provide strong typing on your catch clauses to handle different failure modes differently. So most of them just call the base constructor.